src/extensions/Mention.js

```diff
@@ -0,0 +1,70 @@
+import { mergeAttributes } from '@tiptap/core'
+import TipTapMention from '@tiptap/extension-mention'
+import { generateUrl } from '@nextcloud/router'
+
+const getAvatarUrl = (user, size) => {
+	return generateUrl('/avatar/{user}/{size}', {
+		user,
+		size,
+	})
+};
+
+export default TipTapMention.extend({
+	parseHTML() {
+		return [
+			{
+				tag: `span[data-type="user"]`,
+				getAttrs: element => ((element.getAttribute('data-type') === "user") 
+					&& (element.getAttribute('class') === "mention") 
+					&& null),
+				priority: 100,
+			},
+		]
+	},
+    
+	renderHTML({ node, HTMLAttributes }) {
+		const avatarUrl = getAvatarUrl(node.attrs.id, 44)
+
+		return [
```

Comment:
Is there any way we could use the UserBubble component directly here?

Reply:
To inject `UserBubble` we need to make sure that the `UserBubble` will be handled as a VueJs component.

Reply:
Found a way and pushed it with [fd38108](https://github.com/nextcloud/text/pull/2748/commits/fd3810853a5525b6f2701d82f98829fdd1eec65d)